x-pack/plugins/lens/public/visualizations/xy/add_layer.tsx

```diff
@@ -225,3 +264,40 @@ export function AddLayerButton({
     </>
   );
 }
+
+const ChartOptionWrapper = ({
+  label,
+  description,
+  icon,
+  onClick,
+  type,
+}: {
+  label: string;
+  description: string;
+  icon: IconType;
+  onClick: () => void;
+  type: string;
+}) => {
+  return (
+    <button
+      data-test-subj={`lnsXY_seriesType-${type}`}
+      onClick={onClick}
+      className="euiContextMenuItem lnsLayerAddButton"
+      css={css`
+        padding: 8px 12px;
+        border-bottom: 1px solid #eef2f7;
+        width: 100%;
+        &: hover, &: focus {
+          color: #0071c2;
+          background-color: rgba(0, 119, 204, 0.1);
+          span, .euiText {
+            text-decoration: underline;
+            color: #0071c2;
+          }
+        }
+      `}
+    >
+      <ChartOption option={{ icon, label, description }} />
+    </button>
```

Comment:
This caught my eye because there's a lot of hardcoded color values, which is problematic for dark mode. Out of curiosity, is there a compelling reason to use custom markup and styles here versus using an `EuiSelectable` component again (albeit with the search and selection icons disabled)? I imagine the `EuiSelectable` would be easier, and you wouldn't need to worry about how colors are handled between light/dark mode.

Reply:
I didn't find the way to use EuiSelectable inside the context menu, but I replaced all the hardcoded values with the themed variables. Sorry for it, I forgot to clean it up!